src/pages/landing/LandingPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import landing_image_mellowhead from "../../assets/landing_page_images/landing_page_mellowhead.svg"
+import "./landingPage.css"
+import { Link } from "react-router-dom";
+
+
+export default function LandingPage (){
```

Comment:
exporting the function from here or at the end of the line will have the same effect but if you only exporting one function its good to export from the end as you did in other files it will increase readabity

Reply:
Noted. Thank you!